K2Bridge/Visitors/HistogramVisitor.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
```

Comment:
This is an aggregation visitor, right?
If yes, please move it to the aggregation folder.

Reply:
Done